test-app/tests/integration/components/input-field-test.gts

```diff
@@ -196,4 +196,20 @@ module('Integration | Component | Fields | Input', function (hooks) {
     </template>);
     assert.dom('[data-root-field="selector"]').exists();
   });
+
+  test('it disables the input using `@isDisabled`', async function (assert) {
+    await render(<template>
+      <InputField @label="Label" @isDisabled={{true}} data-input />
+    </template>);
+
+    assert.dom('[data-input]').isDisabled();
+  });
```

Comment:
It looked like we didn't have a disabled test like we do with the other components so I added one